### ISSUE TITLE
Can't download images

### ISSUE THREAD

**Issue by Litoba** [pipoket/booru-grabber#7]
![_2017_02_17_10_38_53_396](https://cloud.githubusercontent.com/assets/15212715/23049139/9e0fc890-f4fd-11e6-889c-c27d749cc71a.png)

thanks you for new version.
check the number of images, but no matter how long it may be, it will not be downloaded.
what seems to be the problem, Is there anything else I need to set up?

**Comment by pipoket** [comment]
No, you should be able to download the file without any additional configuration.

This seems to be a very odd problem. Can you confirm that you can see the image with the browser using "https" connection? (e.g. Browse Gelbooru using https://gelbooru.com URL on any browser).

Also, could you try changing the user agent from default one to any other one? You can see the user agent string of your browser from following website: http://www.whoishostingthis.com/tools/user-agent/ 

(Disclaimer: I am not releated in any way with whoishostingthis website)

**Comment by Litoba** [comment]
Make sure there are two things and try it, but still download is not.
this problem very perplexing.

**Comment by Litoba** [comment]
![_2017_03_02_04_06_44_849](https://cloud.githubusercontent.com/assets/15212715/23476374/ea537e7c-fefd-11e6-939a-1cb2c05d6b3a.png)
Tried to start on another computer.
however can't open due to this problem.
It similar to previous issues errors Looks like.
But it seems need other solutions.

**Comment by pipoket** [comment]
Okay. It might be better to create an debug version which logs everything in detail to solve your issue.
It might take a while, so I'll update here whenever it gets ready.

**Comment by Litoba** [comment]
Thanks a lot for everything! :)
